packages/webapp/public/locales/fr/translation.json

```diff
@@ -524,13 +524,30 @@
       "HEADER": "En-tête",
       "LINK": "Lien",
       "MESSAGE": "Cela supprimera la dépense des registres de votre ferme. Elle ne sera plus reflétée dans aucun résumé financier"
+      "ADD_CUSTOM_EXPENSE": "Ajouter une dépense personnalisée",
+      "ADD_CUSTOM_EXPENSE_TYPE": "Ajouter un type dépense personnalisée",
+      "CUSTOM_EXPENSE_NAME": "Nom de la dépense personnalisée",
+      "CUSTOM_EXPENSE_TYPE": "Type de dépense personnalisée",
+      "DUPLICATE_NAME": "Un type de dépense portant ce nom existe déjà. Veuillez en choisir un autre.",
+      "FLOW": "Création d'une dépense",
+      "MANAGE_CUSTOM_EXPENSE_TYPE": "Gérer les types de dépenses personnalisés",
+      "NEW_EXPENSE_ITEM": "Nouvel article de dépense",
+      "TITLE": "Ajouter une dépense",
+      "WHICH_TYPES_TO_RECORD": "Choisissez vos types de dépenses ci-dessous:"
+    },
+    "CUSTOM_EXPENSE_DESCRIPTION": "Description de la dépense personnalisée",
+    "DELETE": {
+      "CONFIRM": "Confirmer la suppression",
+      "HEADER": "Supprimer la dépense?",
+      "LINK": "Supprimer la dépense",
+      "MESSAGE": "Cela supprimera la dépense des registres de votre ferme. Elle ne sera plus reflétée dans aucun résumé financier."
```

Comment:
Thank you for the PR Loïc!
It seems like line 510 -526 should be updated instead of adding these!

Reply:
Update: I addressed this!